_pytest/test_server.py

```diff
@@ -48,22 +57,41 @@ def test_status(http_server):
 
 
 def test_get_parse(http_server):
-    req = requests.get(http_server + "/parse?q=hello")
-    expected = [{"entities": {}, "confidence": None, "intent": "greet", "_text": "hello"}]
-    assert req.status_code == 200 and req.json() == expected
+    tests = [
+        ResponseTest(
+            "/parse?q=hello",
+            [{u"entities": {}, u"confidence": None, u"intent": u"greet", u"_text": u"hello"}]
+        ),
+        ResponseTest(
+            "/parse?q=hello ńöñàśçií",
+            [{u"entities": {}, u"confidence": None, u"intent": u"greet", u"_text": u"hello ńöñàśçií"}]
+        ),
+    ]
+    for test in tests:
+        req = requests.get(http_server + test.endpoint)
+        assert req.status_code == 200 and req.json() == test.expected_response
 
 
 def test_post_parse(http_server):
-    req = requests.post(http_server + "/parse", json={"q": "hello"})
-    expected = [{"entities": {}, "confidence": None, "intent": "greet", "_text": "hello"}]
-    assert req.status_code == 200 and req.json() == expected
+    tests = [
+        ResponseTest(
+            "/parse",
+            [{u"entities": {}, u"confidence": None, u"intent": u"greet", u"_text": u"hello"}],
+            payload={"q": "hello"}
```

Comment:
I checked the documentation, and actually there we use q...did we change to text at some point? If so I can fix it back in no time :)